lib/LessStringifier.js

```diff
@@ -9,7 +9,13 @@ module.exports = class LessStringifier extends Stringifier {
 
     let name = `${node.raws.identifier || '@'}${node.name}`;
     let params = node.params ? this.rawValue(node, 'params') : '';
-    const important = node.raws.important || '';
+    let important = '';
+
+    if (node.important && node.raws.important) {
```

Comment:
I don't understand this addition. If a node is important, then node.raws.important should always exist?

Reply:
I'll try to explain as good as I can cause this one is a bit funny :)

`postcss` will only include the `!important` part in `node.raws` if it's **not** ` !important` (with a leading space and all lowercase, GitHub removed the leading space). Everything else, a space between `!` and `important`, one or more chars uppercase, etc. will create a `node.raws` property. That's why I made the same check here.